### PR TITLE
fix spelling errors

### PR DESCRIPTION
**Pull request by hjung4**

### CODE REVIEW

File: data/locale/ar.ts [hunk 1]
```diff
@@ -6308,7 +6308,7 @@ Remember to also save your project manually. You can choose to disable saving wh
         <translation type="unfinished"/>
     </message>
     <message>
-        <source>The following errors occured while loading: </source>
+        <source>The following errors occurred while loading: </source>
```
Comment (zonkmachine) [inline data/locale/ar.ts]: The locale files are updated automatically when a string in the source code is changed via our translation project by happy little digital elves. Please revert the changes in `data/locale/`.

File: src/core/Song.cpp [hunk 2]
```diff
@@ -1437,7 +1437,7 @@ QString Song::errorSummary()
 	QString errors = m_errors.join("\n") + '\n';
 
 	errors.prepend( "\n\n" );
-	errors.prepend( tr( "The following errors occured while loading: " ) );
+	errors.prepend( tr( "The following errors occurred while loading: " ) );
```
Comment (zonkmachine) [inline src/core/Song.cpp]: :+1:

File: plugins/lb302/README [hunk 3]
```diff
@@ -15,7 +15,7 @@ BUG:
   to be caused by 'unexhausted buffers' That is, the problem manifests
   itself to a greater degree when the user "buffer size" configuration
   is increased to over 1024 frames or so.  The problem is much less
-  noticable when the buffer size is set to 64 frames.
+  noticeable when the buffer size is set to 64 frames.
```
Comment (zonkmachine) [inline plugins/lb302/README]: :+1:

File: plugins/LadspaEffect/cmt/src/freeverb/readme.txt [hunk 4]
```diff
@@ -13,7 +13,7 @@ I'll try to keep this "readme" reasonably small. There are few things in the wor
...
+"Components" - Contains files that should clean-compile ON ANY TYPE OF COMPUTER OR SYSTEM WHATSOEVER. It should not be necessary to make ANY changes to these files to get them to compile, except to make up for inadequacies of certain compilers. These files create three classes - a comb filter, an allpass filter, and a reverb model made up of a number of instances of the filters, with some features to control the filters at a macro level. You will need to link these classes into another program that interfaces with them. The files in the components drawer are completely independent, and can be built without dependancies on anything else. Because of the simple interface, it should be possible to interface these files to any system - VST, DirectX, anything - without changing them AT ALL.
```
Comment (zonkmachine) [inline plugins/LadspaEffect/cmt/src/freeverb/readme.txt]: Anything in  /`plugins/LadspaEffect/` is actually from various other projects and changes should probably be proposed "upstream". In this case (cmt) as per instructions here: http://www.ladspa.org/cmt/bugs.html

**Comment by hjung4** [comment]
Changes have been made.

**Comment by zonkmachine** [comment]
I think this is good to merge.

File: plugins/lb302/README (hunk 3 above)
Comment (zonkmachine) [inline plugins/lb302/README]: I wouldn't spend too much time on old README files like this though. The rest of the info in there isn't up to date anyway.

**Comment by zonkmachine** [comment]
Merging, thanks!
